RemoveDuplicatedFromSortedArrayII.java

```diff
@@ -0,0 +1,45 @@
+// Time Complexity : O(N)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Yes
```

Comment:
Is this a typo? If not, what problem did you face?

Reply:
Sorry, Sumesh. That was a typo. Please ignore it.